_rules/inline-link-in-paragraph-has-distinguishable-style-not-based-on-color-alone-88407d.md

```diff
@@ -25,19 +25,19 @@ This rule applies to any [visible][] HTML element that is a [semantic link][], f
 
 ## Expectation
 
-For each test target, at least one of the following is true:
+For at least one element of the set comprised by the test target and all of its [visible][] [descendant][] elements in the [flat tree][], at least one of the following is true when the element is neither [focused][] nor [hovered][]:
```

Comment:
I'm not sure that "all of its visible descendants" is good enough:
```html
<p>Lorem <a href="#" class="no-style">A very long link name with <span class="border">a</span> single letter with border</a> Ipsum</p>
```
That would pass the rule. Does that pass the SC? I don't think so…
OTOH
```html
<p>Lorem <a href="#" class="no-style">A <span class="border">very long link name with a single letter without border </span></a> Ipsum</p>
```
could arguably satisfy the SC just because the indication is taking so much of the actual click target that it's OK in virtually all practical cases 🤔

Reply:
Similar to how the text contrast rule accepts a single foreground pixel with sufficient contrast even if _all_ the other foreground pixels have insufficient contrast, I believe this rule is really doing the best it can. Most importantly, it preserves a non-controversial fail-on-fail relation to the success criterion.